src/glmfit.jl

```diff
@@ -58,7 +58,7 @@ function GlmResp(y::V, d::D, l::L, off::V, wts::V) where {V<:FPVector,D,L}
     return r
 end
 
-function GlmResp(y::AbstractVector{<:Real}, d::D, l::L, off::AbstractVector{<:Real}, 
+function GlmResp(y::AbstractVector{<:Real}, d::D, l::L, off::AbstractVector{<:Real},
```

Comment:
can you revert all these whitespace changes so that the git blame is clean?

Reply:
Fixed.